WalletWasabi/Extensions/RPCClientExtensions.cs

```diff
@@ -80,11 +80,7 @@ public static async Task<EstimateSmartFeeResponse> TryEstimateSmartFeeAsync(this
 			if (tryOtherFeeRates)
 			{
 				EstimateSmartFeeResponse response = await rpc.TryEstimateSmartFeeAsync(confirmationTarget, estimateMode);
-				if (response != null)
-				{
-					return response;
-				}
-				else
+				if (response is null)
 				{
 					// Hopefully Bitcoin Core brainfart: https://github.com/bitcoin/bitcoin/issues/14431
 					for (int i = 2; i <= 1008; i++)
```

Comment:
Suggestion: it would be good to have this constant 1008 (confirmation target) in the `Constants` static class. This is not the only place where this magic number is.

Reply:
ACK

Reply:
Done. #1720